app/lib.test.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs')
+const path = require('path')
+const {
+  decryptVault,
+  extractVaultFromFile,
+} = require('./lib.js')
+
+const crypto = require('crypto');
+
+Object.defineProperty(globalThis, 'crypto', {
```

Comment:
Instead of modifying the `crypto` module directly, can we use `jest.mock` to mock it out?

Reply:
I was not able to make it work using `jest.mock` - and I also think that mocking is not correct here as we still want to use the runtime module as-is.

I went with moving this part out, as suggested by [jest docs](https://jestjs.io/docs/configuration#globals-object).

1821fa532bc600a3546a071a33d665c2d727ea9f